assets/src/application/ui/input/InlineEditInput/InlineEditHeading.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import Editable from './Editable';
+import { HeadingProps } from './types';
+
+const InlineEditHeading: React.FC<HeadingProps> = ({ ...rest }) => {
+	return <Editable inputType='heading' level={4} {...rest} />;
```

Comment:
what about:
```
const level = rest.level || 4;
<Editable {...rest} inputType='heading' level={level}  />;
```
?
this way there is no guessing

Reply:
I don't think that makes it any different.